rust/operator-binary/src/controller.rs

```diff
@@ -103,6 +103,7 @@ pub struct Ctx {
 
 pub const OPERATOR_NAME: &str = "trino.stackable.tech";
 pub const CONTROLLER_NAME: &str = "trinocluster";
+pub const FULL_CONTROLLER_NAME: &str = "trinocluster.trino.stackable.tech";
```

Comment:
note: We should combine already existing const values here instead. Please take a look at [const_format](https://docs.rs/const_format/latest/const_format/), more specifically [concatcp](https://docs.rs/const_format/latest/const_format/macro.concatcp.html).

Reply:
I intentionally decided against `const_format` to avoid adding a dependecy to all our operators. But no strong opinion, so changed it in f8f20d4